requirements.txt

```diff
@@ -1,2 +1 @@
-IPython
```

Comment:
`Ipython` and `streamlit` may could be added to `setup` as `extras_require` option.